crates/typos-dict/assets/words.csv

```diff
@@ -28613,7 +28613,6 @@ havve,have
 hax,hex
 haynus,heinous
 hazzle,hassle
-hd,had
```

Comment:
Please add it to `allowed.csv` with an explanation.  This will prevent it from being added back in the future.